### PR TITLE
Apply internal changes on top of Github's version

### PR DESCRIPTION
**Pull request by lucagiovagnoli**
I just applied a patch on topI removed:
- references to our internal nexus in the pom file
- reference to internal issue numbers in comments

For review @talalryz @farrell  @rvogan

### CODE REVIEW

File: src/main/java/biz/k11i/xgboost/tree/PreorderRegTree.java [hunk 1]
```diff
@@ -7,7 +7,6 @@
 import biz.k11i.xgboost.util.FVec;
 
 /**
- * author: adamj, niloyg
```
Comment (talalryz) [inline src/main/java/biz/k11i/xgboost/tree/PreorderRegTree.java:10]: why remove author line?

**Review by talalryz — APPROVED** [review]
lgtm! I couldn't find any references to internal yelp stuff

**Comment by talalryz** [comment]
It seems that the build failed on oracle jdk8

File: src/main/java/biz/k11i/xgboost/tree/PreorderRegTree.java (hunk 1 above)
Comment (lucagiovagnoli) [reply src/main/java/biz/k11i/xgboost/tree/PreorderRegTree.java:10]: I didn't write that, I'm just running `git apply`. Not sure why they wanted to remove it

**Review by cpfarrell — APPROVED** [review]
Looks good to me

**Comment by lucagiovagnoli** [comment]
I removed oracklejdk8 tests since travis doesn't install it on xenial anymore: https://github.com/travis-ci/travis-ci/issues/10290

**Review by talalryz — APPROVED** [review]

**Pull request merged**